### PR TITLE
Cleaned up "TRL <date>:" comments in code.

### PR DESCRIPTION
**Pull request by tylerrleblond**

### CODE REVIEW

File: src/layout/dynamic_layouts/edpc_layout.cpp [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+    // Print out the grid for the purposes of debugging
+    // std::cout << "The number of rows in the grid is: " << grid.size() << std::endl;
+    // std::cout << "The number of columns in the grid is: " << grid[grid.size()-1].size() << std::endl;
+    // for (unsigned int i=0; i<grid.size(); i++) {
+    //     for (unsigned int j=0; j<grid[i].size(); j++) {
+    //         std::cout << grid[i][j] << ' ';
+    //     }
+    //     std::cout << std::endl;
+    // }
```
Comment (gwwatkin) [inline src/layout/dynamic_layouts/edpc_layout.cpp:123]: Could do something like this:

```suggestion
#ifdef DEBUG_EDPC_LAYOUT_CREATION
    // Print out the grid for the purposes of debugging
    // std::cout << "The number of rows in the grid is: " << grid.size() << std::endl;
    // std::cout << "The number of columns in the grid is: " << grid[grid.size()-1].size() << std::endl;
    // for (unsigned int i=0; i<grid.size(); i++) {
    //     for (unsigned int j=0; j<grid[i].size(); j++) {
    //         std::cout << grid[i][j] << ' ';
    //     }
    //     std::cout << std::endl;
    // }
#endif
```

**Review by gwwatkin — COMMENTED** [review]
`compact_demo_1.spec.new` needs to be removed

File: src/patches/dense_slice.cpp [hunk 2]
```diff
@@ -128,10 +127,15 @@ DenseSlice::DenseSlice(const lsqecc::Layout &layout, const tsl::ordered_set<Patc
...
+    
     size_t distillation_time_offset = 0;
     for(auto t : layout.distillation_times())
-        time_to_next_magic_state_by_distillation_region.push_back(t+distillation_time_offset++);
+        if (edpclayout) {
+            time_to_next_magic_state_by_distillation_region.push_back(t);
+        }
+        else {
+            time_to_next_magic_state_by_distillation_region.push_back(t+distillation_time_offset++);           
+        }
```
Comment (gwwatkin) [inline src/patches/dense_slice.cpp:138]: Would be nice to have this as a separate option. But good for a separate PR

Comment (tylerrleblond) [reply src/patches/dense_slice.cpp]: Added it as a separate option in the newest commit.

File: src/layout/dynamic_layouts/edpc_layout.cpp (hunk 1 above)
Comment (tylerrleblond) [reply src/layout/dynamic_layouts/edpc_layout.cpp]: Made this change in the newest commit.

**Comment by gwwatkin** [comment]
superseded by #41

**Pull request closed without merging**